docs/static/dead-letter-queues.asciidoc

```diff
@@ -250,3 +264,17 @@ output {
 <3> The clean event is sent to Elasticsearch, where it can be indexed because
 the mapping issue is resolved.
 
+[[dlq-size]]
+==== Track dead letter queue size
+
+ToDo: Add info on how to track size before it becomes a problem
```

Comment:
The size of the DLQ for each pipeline that has it enabled is available in the node stats API at the address:
`pipelines.${pipeline_id}.dead_letter_queue.queue_size_in_bytes`.